db/data/regions/AT/en.yml

```diff
@@ -5,13 +5,15 @@ en:
       addresses:
         address1:
           label:
-            default: Street and house number
-            optional: Street and house number (optional)
+            default: Province
+            optional: Province (optional)
```

Comment:
What's the context for this changing?

- this is address line 1
- Austria has states, not provinces (and they don't need to be included in the address)

Reply:
Ahh that's a mistake, good catch